scss/bitstyles/components/button-groups.stories.mdx

```diff
@@ -88,29 +87,97 @@ For a series of buttons that are very directly related e.g.
+              <path d="M67.17,2.78a6.07,6.07,0,0,1,8.59,8.58h0L41.41,45.71a6.07,6.07,0,0,0,0,8.58L75.76,88.64a6.07,6.07,0,1,1-8.59,8.58L24.24,54.29a6.07,6.07,0,0,1,0-8.58Z" fill-rule="evenodd"/>
+            </svg>
+            <span class="u-sr-only">previous</span>
+          </button>
+        </li>
+        <li class="l-flex">
+          <button class="o-button o-button--icon o-button--grouped o-button--grouped__last" type="button">
+            <svg class="o-button__icon o-icon o-icon--m" viewBox="0 0 100 100" width="20" height="20" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
+              <path d="M32.83,97.22a6.07,6.07,0,1,1-8.59-8.58L58.59,54.29a6.07,6.07,0,0,0,0-8.58L24.24,11.36a6.07,6.07,0,1,1,8.59-8.58L75.76,45.71a6.07,6.07,0,0,1,0,8.58Z" fill-rule="evenodd"/>
+            </svg>
+            <span class="u-sr-only">next</span>
+          </button>
+        </li>
+      </ul>
+    `}
+  </Story>
+  <Story name="Joined buttons - pagination">
+    {`
+      <ul class="o-list-reset l-flex l-flex--inline l-items-stretch o-button--grouped-container">
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped o-button--grouped__first">
+            <svg class="o-button__icon o-icon o-icon--m" viewBox="0 0 100 100" width="20" height="20" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
+              <path d="M67.17,2.78a6.07,6.07,0,0,1,8.59,8.58h0L41.41,45.71a6.07,6.07,0,0,0,0,8.58L75.76,88.64a6.07,6.07,0,1,1-8.59,8.58L24.24,54.29a6.07,6.07,0,0,1,0-8.58Z" fill-rule="evenodd"/>
+            </svg>
+            <span class="u-sr-only">previous</span>
+          </a>
+        </li>
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped u-h6">
+            1
+          </a>
+        </li>
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped u-h6">
+            2
+          </a>
+        </li>
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped u-h6">
+            3
+          </a>
+        </li>
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped u-h6">
+            4
+          </a>
+        </li>
+        <li class="l-flex">
+          <a href="/" class="o-button o-button--icon o-button--grouped o-button--grouped__last">
+            <svg class="o-button__icon o-icon o-icon--m" viewBox="0 0 100 100" width="20" height="20" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
+              <path d="M32.83,97.22a6.07,6.07,0,1,1-8.59-8.58L58.59,54.29a6.07,6.07,0,0,0,0-8.58L24.24,11.36a6.07,6.07,0,1,1,8.59-8.58L75.76,45.71a6.07,6.07,0,0,1,0,8.58Z" fill-rule="evenodd"/>
+            </svg>
+            <span class="u-sr-only">next</span>
+          </a>
+        </li>
+      </ul>
+    `}
+  </Story>
 </Canvas>
 
+## Modal buttons
```

Comment:
Are these also called `Modal`? In my mind, that name is firmly enmeshed with the large, centered dialogs on dark, partially-transparent backdrops that block editing. 😕

Reply:
Yeh I get it :D Me too, to some extent. These are modal in that they switch between & indicate modes. I’ll think about it, but maybe `mode buttons` is a clear enough distinction?